modprobe.conf/modprobe.conf.fs-blacklist

```diff
@@ -0,0 +1,32 @@
+# These file systems are not used on most systems but the kernel
+# will still happily load the modules for them when presented with
+# a potentially malicious device like a USB stick.  Many of these
+# have not been maintained except for in-kernel API compatibility
+# in many years, have not been audited, and may have security vulnerabilities.
+#
+# The following list only specifies local file systems since those are the
+# only ones that will be detected automatically by mount(8).
+#
+# Enable at your own risk.
+#
+blacklist adfs
+blacklist affs
+blacklist bfs
+blacklist befs
+blacklist cramfs
+blacklist efs
+blacklist erofs
+blacklist exofs
+blacklist freevxfs
+blacklist f2fs
+blacklist hfs
+blacklist hpfs
+blacklist jffs2
```

Comment:
This is a raw flash filesystem, it does not work on a block device. So there is no way to have this automatically mounted by plugging in a USB thumb drive.

Reply:
True, and the code doesn't allow loopback mounts.  We can drop jffs2.